R/bpmnVisualizationR.R

```diff
@@ -55,17 +55,19 @@
 #' )
 #'
 #' overlays <- list(
-#'   bpmnVisualizationR::create_overlay("start_event_1_1", "42"),
-#'   bpmnVisualizationR::create_overlay("sequence_flow_1_1", "42", flowStyle),
-#'   bpmnVisualizationR::create_overlay("task_1_1", "9", taskStyle)
+#'   bpmnVisualizationR::create_overlay("start_event_1_1", "42", position = "middle-left"),
+#'   bpmnVisualizationR::create_overlay("sequence_flow_1_1", "42", flowStyle, "end"),
+#'   bpmnVisualizationR::create_overlay("task_1_1", "9", taskStyle),
+#'   bpmnVisualizationR::create_overlay("sequence_flow_1_2", "8")
```

Comment:
**question:** can we add an overlay without any style and position on the "Approve Order" call activity to show how it behaves? If so, add it also to the next example to show the difference when `enableDefaultOverlayStyle` is disabled.